### ISSUE TITLE
Documentation needed for uniqueString parameter

### ISSUE THREAD

**Issue by JeffGiroux** [F5Networks/f5-aws-cloudformation-v2#9]
## Do you already have an issue opened with F5 support?
GitHub Issues are consistently monitored by F5 staff, but should be considered as best-effort only and you should not expect to receive the same level of response as provided by F5 Support. Please [open a case](https://support.f5.com/csp/article/K2633) with F5 if this is a critical issue.

## Description
There is a parameter called uniqueString in the parent template that passes onto the linked templates. It is unclear what this parameter is for as it is not documented in the INPUT parameters section. Please document. I had to dig through the modules to find the bigip-standalone variable containing the info.

At high level, it sounds like a prefix.

```
  uniqueString:
    AllowedPattern: '^[a-zA-Z][a-zA-Z0-9]{1,11}$'
    ConstraintDescription: Must Contain between 1 and 12 alphanumeric characters with
      first character as a letter.
    Default: myUniqStr
    Description: Unique String used when creating object names or Tags.
    Type: String
```

Based on the bigip module code, here is the usage of that value as a prefix. If this is the intent, please update parent example README with the input parameters.

bigip-standalone.yaml module using the uniqueString as prefix
```
        - Key: Name
          Value: !Join
            - ''
            - - !Ref 'uniqueString'
              - '-bigip-instance'
```

## Template
[For bugs, enter the template with which you are experiencing issues below.
](https://github.com/F5Networks/f5-aws-cloudformation-v2/tree/main/examples/quickstart)

## Severity Level
3

**Comment by G-gonzalezjimenez** [comment]
Thanks for the comment, tracking internally with ESECLDTPLT-2984

**Comment by shyawnkarim** [comment]
Closing. 

This ticket was included with [Release 2.2.0.0](https://github.com/F5Networks/f5-aws-cloudformation-v2/releases/tag/v2.2.0.0).
